<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NPCGeneratorWidget</class>
 <widget class="QWidget" name="NPCGeneratorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NPC Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QComboBox" name="comboGender">
       <item>
        <property name="text">
         <string>female</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>male</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRace">
       <property name="text">
        <string>R&amp;ace:</string>
       </property>
       <property name="buddy">
        <cstring>comboRace</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboSpecies"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSpecies">
       <property name="text">
        <string>&amp;Species:</string>
       </property>
       <property name="buddy">
        <cstring>comboSpecies</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelGender">
       <property name="text">
        <string>G&amp;ender:</string>
       </property>
       <property name="buddy">
        <cstring>comboGender</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboRace"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelRegion">
       <property name="text">
        <string>&amp;Region:</string>
       </property>
       <property name="buddy">
        <cstring>comboRegion</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboRegion"/>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkRandomRegion">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="checkRandomSpecies">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="checkRandomRace">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkRandomGender">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonGenerate">
     <property name="text">
      <string>&amp;Generate</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>lineName</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelHairColor">
       <property name="text">
        <string>Hair colour:</string>
       </property>
       <property name="buddy">
        <cstring>lineHairColor</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelEyeColor">
       <property name="text">
        <string>Eye colour:</string>
       </property>
       <property name="buddy">
        <cstring>lineEyeColor</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineName"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineHairColor"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEyeColor"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
       <property name="buddy">
        <cstring>lineSize</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineSize"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelWeight">
       <property name="text">
        <string>Weight:</string>
       </property>
       <property name="buddy">
        <cstring>lineWeight</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineWeight"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textResult"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
